beacon-chain/sync/initial-sync/service.go

```diff
@@ -117,6 +118,7 @@ func (s *Service) Start() {
 			genesis,
 		).Warn("Genesis time is in the future - waiting to start sync...")
 		time.Sleep(roughtime.Until(genesis))
+		logutil.CountdownToGenesis(genesis, 300)
```

Comment:
What is 300? We call this a "magic number".

Please define a constant or use an inline comment like:
```suggestion
		logutil.CountdownToGenesis(genesis, 300 /* countdown interval */)
```

Reply:
Does this work at runtime? Just 1 line above you have a blocking call that sleeps until genesis time.

Reply:
Okay will put the comment 
So I have to put it before it?